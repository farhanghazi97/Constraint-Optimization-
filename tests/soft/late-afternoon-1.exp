m1:mon 4pm
cost:0
