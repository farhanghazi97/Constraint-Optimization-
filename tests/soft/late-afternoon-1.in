# Testing late afternoon constraints
meeting, m1
domain, m1, late-afternoon, soft
