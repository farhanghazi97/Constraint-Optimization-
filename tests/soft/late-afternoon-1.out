m1:fri 4pm
cost:0
